executor/select_into_test.go

```diff
@@ -19,73 +19,77 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/executor"
 	"github.com/pingcap/tidb/parser/mysql"
+	"github.com/pingcap/tidb/testkit"
 	"github.com/pingcap/tidb/types"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/stretchr/testify/require"
 )
 
-func cmpAndRm(expected, outfile string, c *C) {
+func cmpAndRm(expected, outfile string, t *testing.T) {
 	content, err := os.ReadFile(outfile)
-	c.Assert(err, IsNil)
-	c.Assert(string(content), Equals, expected)
-	c.Assert(os.Remove(outfile), IsNil)
+	require.NoError(t, err)
+	require.Equal(t, expected, string(content))
+	require.Nil(t, os.Remove(outfile))
 }
 
 func randomSelectFilePath(testName string) string {
 	return filepath.Join(os.TempDir(), fmt.Sprintf("select-into-%v-%v.data", testName, time.Now().Nanosecond()))
 }
 
-func (s *testSuite1) TestSelectIntoFileExists(c *C) {
+func TestSelectIntoFileExists(t *testing.T) {
 	outfile := randomSelectFilePath("TestSelectIntoFileExists")
 	defer func() {
-		c.Assert(os.Remove(outfile), IsNil)
+		require.Nil(t, os.Remove(outfile))
```

Comment:
```os.Remove``` return error. so it should use ```require.NoError```.

ditto other.

Reply:
ok fixed